x-pack/plugin/sql/src/main/java/org/elasticsearch/xpack/sql/expression/function/scalar/datetime/DateFormatter.java

```diff
@@ -100,7 +100,7 @@ private static String ordinalSuffixForDayOfTheMonth(int i) {
     private Function<TemporalAccessor, String> javaFormat;
     private Function<String, String> additionalMapper;
 
-    public static Function<TemporalAccessor, String> ofPattern(String dateFormatPattern) {
+    protected static Function<TemporalAccessor, String> ofPattern(String dateFormatPattern) {
```

Comment:
I had a different change in mind: **no** access modifiers, meaning the default one ("package protected").

Reply:
Yes of course